packages/linter/src/caches.ts

```diff
@@ -0,0 +1,14 @@
+import { existsSync, readFileSync } from "fs";
+import ampCacheList from "amp-toolbox-cache-list";
```

Comment:
nit: s/ampCacheList/AmpCacheList

Reply:
Changed in 2cd1d6aef7855320a07a5a6f03e39e9ff2cbff11.